krabs/krabs/filtering/event_filter.hpp

```diff
@@ -49,6 +50,14 @@ namespace krabs {
          */
         event_filter(filter_predicate predicate);
 
+        /**
```

Comment:
here and in other files, please ensure that you are indenting with spaces not tabs. please indent by four spaces.

Reply:
Done